src/core/loss_detection.c

```diff
@@ -1332,6 +1332,14 @@ QuicLossDetectionProcessAckBlocks(
         // which would mean we mistakenly classified those packets as lost.
         //
         if (*LostPacketsStart != NULL) {
+            QUIC_SENT_PACKET_METADATA *lastNode =
+                CXPLAT_CONTAINING_RECORD(
+                    LossDetection->LostPacketsTail, QUIC_SENT_PACKET_METADATA,
+                    Next);
+
+            if (lastNode->PacketNumber < AckBlock->Low) {
+                goto CheckSentPackets;
+            }
```

Comment:
```suggestion
            QUIC_SENT_PACKET_METADATA *lastNode =
                CXPLAT_CONTAINING_RECORD(
                    LossDetection->LostPacketsTail, QUIC_SENT_PACKET_METADATA,
                    Next);
            if (lastNode->PacketNumber < AckBlock->Low) {
                goto CheckSentPackets;
            }

```
nit: my personal preference on style

Reply:
Done changes.